roles/common/defaults/main.yaml

```diff
@@ -1,21 +1,25 @@
 ---
 common_debug_massage: "hElLo tHeRe"
 
-# NTP Role
-ntp_package: ntp
-ntp_service: ntpd
-ntp_configfile: /etc/ntp.conf
-ntp_configfile_user: root
-ntp_configfile_group: root
-ntp_timezone: Europe/Kiev
+# Packages that would be installed
+package_list:
+  - ntp
+
+# Default Timezone
+ntp_timezone:
```

Comment:
Interesting solution. But, I believe that it is better to specify one default value. Or do you have an explanation for why you specified such a list of timezones? Why are there only 4?

Reply:
i just trying to understand syntax and what i can do in code and what doesn't. i will remove superfluous parameters